x/upgrade/keeper/keeper.go

```diff
@@ -100,19 +107,45 @@ func (k Keeper) SetUpgradedClient(ctx sdk.Context, cs ibcexported.ClientState) e
 	}
 
 	store := ctx.KVStore(k.storeKey)
-	store.Set(types.UpgradedClientKey(), bz)
+	store.Set(types.UpgradedClientKey(upgradeHeight), bz)
 	return nil
 }
 
 // GetUpgradedClient gets the expected upgraded client for the next version of this chain
-func (k Keeper) GetUpgradedClient(ctx sdk.Context) (ibcexported.ClientState, error) {
+// along with the planned upgrade height
+func (k Keeper) GetUpgradedClient(ctx sdk.Context) (ibcexported.ClientState, int64, error) {
```

Comment:
is there tests for this func?

Reply:
Implicitly it's used in the upgrade tests, but I can create explicit tests